README.rst

```diff
@@ -1,14 +1,14 @@
 |Build Status| |Coverage Status|
 
-Patroni: A Template for PostgreSQL HA with ZooKeeper, etcd or Consul
+Patroni: Template for PostgreSQL HA with Kubernetes, ZooKeeper, etcd or Consul
```

Comment:
Somebody might think that Kubernetes is some strange animal similar to ZooKeeper. Very confusing :(